src/main/kotlin/no/nav/syfo/persistering/HandleOKOppgave.kt

```diff
@@ -74,3 +82,22 @@ suspend fun handleOKOppgave(
         log.info("Hopper over ferdigstillOppgave, oppgaveId $oppgaveId er allerede ${oppgave.status}")
     }
 }
+
+private fun shouldCreateOppfolgingsOppgave(receivedSykmelding: ReceivedSykmelding): Boolean {
+    return receivedSykmelding.merknader?.isNotEmpty() == true
+}
+
+fun createOppfolgingsoppgave(receivedSykmelding: ReceivedSykmelding, enhet: String, veileder: Veileder): Oppgave =
+    Oppgave(
+        aktoerId = receivedSykmelding.sykmelding.pasientAktoerId,
+        opprettetAvEnhetsnr = "9999",
+        tildeltEnhetsnr = "4488", // Sendes til NAY v/ 4488
```

Comment:
Generelt er det fy fy å hardkode inn enhetsnummer. Dette skal egentlig løses av rutingregler sånn at man slipper å huske på hvor dette er hardkodet og endre det hvis man bytter enhetsnummer for NAY.

Reply:
Ah, okay. Da er jeg litt usikker på hvordan dette løses best? Er det GOSYS som ev. må implementere disse for oss?

Reply:
På dagens statusmøte ble vi enig om at dette løses best med routingregler, og at de består i å sette:

`        tema = "SYM",
        oppgavetype = "BEH_EL_SYM",
        behandlingstype = "ae0239",`

Dette er nå implementert, men vi er avhengige av at Gosys endrer funksjonalitet for åpning av syfosmmanuell før vi kan rulle dette ut. 

I praksis innebærer dette at oppgaven må vente til dette er på plass, forhåpentligvis etter påske en gang.